src/providers/SubscriptionProvider/SubscriptionProvider.container.js

```diff
@@ -67,9 +67,12 @@ export class SubscriptionProvider extends Component {
   };
 
   configPurchaseValidator = () => {
+    const transformReceipt = this.transformReceipt;
```

Comment:
The only use of tranformReceipt is inside configPurchaseValidator function. Declare the function tranformReceipt inside of this scope and avoid the use of 'this'.